services/graph/pkg/service/v0/graph.go

```diff
@@ -76,6 +76,9 @@ type Graph struct {
 
 // ServeHTTP implements the Service interface.
 func (g Graph) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Adding this as a fix for some weird unescaping issues we've had.
```

Comment:
Could you please add some more info here what the issue actually is. (With a link to the chi-router github issue?)

Reply:
Also: I think this fixes some open issue(s), could you please mention them with `Fixes: <#issue>` in the commit message?

Reply:
Ah, yes, should have done that. Fixed now.